src/melee.cpp

```diff
@@ -1369,14 +1393,6 @@ bool player::block_hit(Creature *source, body_part &bp_hit, damage_instance &dam
     // So that we don't suddenly switch that for any reason.
     const bool weapon_blocking = can_weapon_block();
```

Comment:
`can_weapon_block` only returns true if you can block with a weapon (not a shield), this will need changing. I think I have an idea, though.